appeals/web/src/server/appeals/appeal-details/manage-linked-appeals/manage-linked-appeals.mapper.js

```diff
@@ -62,17 +53,25 @@ export function manageLinkedAppealsPage(appealData, relationshipId, appealId, pa
 				rows: [
 					[
 						{
-							html: `<a class="govuk-link" href="/appeals-service/appeal-details/${
-								appealData.appealId
-							}" aria-label="Appeal ${numberToAccessibleDigitLabel(
-								appealShortReference(appealData.appealReference) || ''
-							)}">${appealShortReference(appealData.appealReference)}</a>`
+							html: isChildOfHorizonAppeal
```

Comment:
Not required but ternaries in ternaries can be hard to read, should this be broken in to a function?